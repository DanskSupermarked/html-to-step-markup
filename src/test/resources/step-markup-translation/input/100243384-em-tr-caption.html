<p><b>Semper tilskudsblanding fra 6 måneder</b></p>  <p>Semper Allomin 2 pulver tilskudsblanding</p><p>Denne tilskudsblanding bruges som supplement eller erstatning for modermælk som del af en varieret kost til børn over 6 måneder.</p> <p><strong>Allomin 2</strong> indeholder en udvalgt blanding af mælkefedt og vegetabilsk olie, som vi kalder Lipilact. Blandingen indeholder flere komponenter, som findes i mælkefedtdråbernes membraner (MFGM), eksempelvis fosfolipider, gangliosider og glykoproteiner.</p> <p>Jern bidrager til en normal kognitiv udvikling hos børn. D-vitamin er nødvendigt for normal vækst og udvikling af knogler. Jod bidrager til normal vækst hos børn.<br /> Derfor er Allomin 2 beriget med jern, D-vitamin og jod i overensstemmelse med den danske lovgivning samt indeholder GOS (galactooligosaccharider).<br /> Jernindholdet i Allomin 2 er 1,2 mg/100 ml færdigt produkt.</p> <p>Pakken indeholder 800 g (2x400 g) pulver ~ ca. 5,4 l færdigblandet modermælkserstatning.</p> <p><b>Vigtigt!</b> Modermælk er den bedste ernæring til spædbørn. Allomin 2 kan ikke bruges som erstatning for modermælk i barnets første 6 måneder. Spørg altid sundhedsplejersken til råds om barnets mad.</p> <p><b>Hvordan gør jeg?</b></p> <table width="300" cellspacing="1" cellpadding="1" border="1">     <caption>Doseringstabel</caption>     <tbody>         <tr>             <td>Vand</td>             <td>Pulver</td>             <td>Færdig mængde, ca.</td>         </tr>         <tr>             <td>60 ml</td>             <td>2 strøgne mål</td>             <td>65 ml</td>         </tr>         <tr>             <td>90 ml</td>             <td>3&nbsp;strøgne mål</td>             <td>100 ml</td>         </tr>         <tr>             <td>120 ml</td>             <td>4&nbsp;strøgne mål</td>             <td>135 ml</td>         </tr>         <tr>             <td>150 ml</td>             <td>5&nbsp;strøgne mål</td>             <td>165 ml</td>         </tr>         <tr>             <td>180 ml</td>             <td>6&nbsp;strøgne mål</td>             <td>200 ml</td>         </tr>         <tr>             <td>210 ml</td>             <td>7 strøgne mål</td>             <td>235 ml</td>         </tr>         <tr>             <td>240 ml</td>             <td>8&nbsp;strøgne mål</td>             <td>265 ml</td>         </tr>     </tbody> </table> <p><b>ANVEND ALTID MEDFØLGENDE MÅLESKE!</b><br /> Det er vigtigt, at du bruger den korrekte mængde pulver i forhold til det afmålte vand. For meget pulver kan give forstoppelse. For lidt pulver kan medføre, at barnet får for lidt energi og næring.</p> <p><em>1 strøget mål ~ ca. 4,9 g pulver</em></p> <p>Vask altid hænder inden du tilbereder Allomin 2.<br /> Sørg for, at sutten og flasken er rene.<br /> Mål vandet op i sutteflasken.<br /> Tilsæt det korrekte antal strøgne mål pulver.<br /> Luk flasken og ryst til pulveret er opløst.<br /> Varm uden låg og sut ved fuld i effekt i mikrobølgeovn i ca. 40 sekunder.<br /> Tjek temperaturen ved at dryppe lidt ud på indersiden af håndleddet. Smag selv, inden du giver det til barnet.<br /> Serv&eacute;r så frisklavet som muligt. Eventuelle rester kan ikke gemmes.</p> <p>Kan også tilberedes med vand fra den kolde hane. <em>OBS! Tilberedes flere flasker ad gangen, skal det altid ske på basis af kogt vand.</em></p>
