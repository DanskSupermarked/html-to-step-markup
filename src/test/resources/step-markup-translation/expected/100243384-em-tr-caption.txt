<bold>Semper tilskudsblanding fra 6 måneder</bold><return/>
<return/>
Semper Allomin 2 pulver tilskudsblanding<return/>
<return/>
Denne tilskudsblanding bruges som supplement eller erstatning for modermælk som del af en varieret kost til børn over 6 måneder.<return/>
<return/>
<bold>Allomin 2</bold> indeholder en udvalgt blanding af mælkefedt og vegetabilsk olie, som vi kalder Lipilact. Blandingen indeholder flere komponenter, som findes i mælkefedtdråbernes membraner (MFGM), eksempelvis fosfolipider, gangliosider og glykoproteiner.<return/>
<return/>
Jern bidrager til en normal kognitiv udvikling hos børn. D-vitamin er nødvendigt for normal vækst og udvikling af knogler. Jod bidrager til normal vækst hos børn.<return/>
Derfor er Allomin 2 beriget med jern, D-vitamin og jod i overensstemmelse med den danske lovgivning samt indeholder GOS (galactooligosaccharider).<return/>
Jernindholdet i Allomin 2 er 1,2 mg/100 ml færdigt produkt.<return/>
<return/>
Pakken indeholder 800 g (2x400 g) pulver ~ ca. 5,4 l færdigblandet modermælkserstatning.<return/>
<return/>
<bold>Vigtigt!</bold> Modermælk er den bedste ernæring til spædbørn. Allomin 2 kan ikke bruges som erstatning for modermælk i barnets første 6 måneder. Spørg altid sundhedsplejersken til råds om barnets mad.<return/>
<return/>
<bold>Hvordan gør jeg?</bold><return/>
<italic>Doseringstabel</italic><return/>
Vand | Pulver | Færdig mængde, ca.<return/>
60 ml | 2 strøgne mål | 65 ml<return/>
90 ml | 3 strøgne mål | 100 ml<return/>
120 ml | 4 strøgne mål | 135 ml<return/>
150 ml | 5 strøgne mål | 165 ml<return/>
180 ml | 6 strøgne mål | 200 ml<return/>
210 ml | 7 strøgne mål | 235 ml<return/>
240 ml | 8 strøgne mål | 265 ml<return/>
<bold>ANVEND ALTID MEDFØLGENDE MÅLESKE!</bold><return/>
Det er vigtigt, at du bruger den korrekte mængde pulver i forhold til det afmålte vand. For meget pulver kan give forstoppelse. For lidt pulver kan medføre, at barnet får for lidt energi og næring.<return/>
<return/>
<italic>1 strøget mål ~ ca. 4,9 g pulver</italic><return/>
<return/>
Vask altid hænder inden du tilbereder Allomin 2.<return/>
Sørg for, at sutten og flasken er rene.<return/>
Mål vandet op i sutteflasken.<return/>
Tilsæt det korrekte antal strøgne mål pulver.<return/>
Luk flasken og ryst til pulveret er opløst.<return/>
Varm uden låg og sut ved fuld i effekt i mikrobølgeovn i ca. 40 sekunder.<return/>
Tjek temperaturen ved at dryppe lidt ud på indersiden af håndleddet. Smag selv, inden du giver det til barnet.<return/>
Servér så frisklavet som muligt. Eventuelle rester kan ikke gemmes.<return/>
<return/>
Kan også tilberedes med vand fra den kolde hane. <italic>OBS! Tilberedes flere flasker ad gangen, skal det altid ske på basis af kogt vand.</italic>
